client/client.go

```diff
@@ -2834,7 +2834,7 @@ func (c *Client) setGaugeForMemoryStats(nodeID string, hStats *stats.HostStats,
 // setGaugeForCPUStats proxies metrics for CPU specific statistics
 func (c *Client) setGaugeForCPUStats(nodeID string, hStats *stats.HostStats, baseLabels []metrics.Label) {
 	for _, cpu := range hStats.CPU {
-		labels := append(baseLabels, metrics.Label{
+		labels := append(baseLabels, metrics.Label{ //nolint:gocritic
```

Comment:
Is the issue here is that the linter is warning about `baseLabels` being modified?

Reply:
Yep. It's warning because the append assignment identifier is not the same as the first argument to append(). It therefore fails equality checking done by [go-toolsmith/astequal](https://github.com/go-toolsmith/astequal).

Reply:
Again I think we should just explicitly copy baseLabels to labels. That's what this code does implicitly, so I think it's best to make that explicit and fix the lint.